docs/docusaurus/docs/cloud/connect/connect_snowflake.md

```diff
@@ -25,56 +23,74 @@ New to GX Cloud and not sure that it's the right solution for your organization?
+    
+        To locate your Snowflake organization name, account name, or legacy account locator values see [Finding the Organization and Account Name for an Account](https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-organization-and-account-name-for-an-account) or [Using an Account Locator as an Identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier#using-an-account-locator-as-an-identifier).
+
+    - **Password**: Enter a Snowflake password. To improve data security, GX recommends using a Snowflake service account to connect to GX Cloud.
 
-2. Copy and paste the following code into the SQL worksheet:
+    - **Database**: Enter the name of the Snowflake database where the data you want to validate is stored. In Snowsight, click **Data** > **Databases**. In the Snowflake Classic Console, click **Databases**.
+ 
+    - **Schema**: Enter the name of the Snowflake schema (table) where the data you want to validate is stored.
 
-   ```sh title="SQL worksheet"
-   use role accountadmin;
-   create user gx_user password="secure_password";
-   create role gx_role;
-   grant role gx_role to user gx_user;
-   ```
-3. Replace `secure_password` with your value.
+    - **Warehouse**: Enter the name of your Snowflake database warehouse. In Snowsight, click **Admin** > **Warehouses**. In the Snowflake Classic Console, click **Warehouses**.
 
-4. Select **Run All** to define your user password, create a new GX role (`gx_role`), and assign the password and role to a new user (`gx_user`).
+    - **Role**: Enter your Snowflake role.
 
-    ![Snowflake Run All](/img/run_all.png)
+8. Optional. Select **Test connection** to test the Data Source connection. Testing the connection to the Data Source is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
 
-5. Copy the following code and paste it into the SQL worksheet:
+9. Click **Continue**.
 
-   ```sh title="SQL worksheet"
-   create warehouse gx_wh
-   warehouse_size=xsmall 
-   auto_suspend=10  
-   auto_resume=true
-   initially_suspended=true;
-   ```
-    The settings in the code example optimize cost and performance. Adjust them to suit your business requirements.
+10. Select **Table Asset** or **Query Asset** and complete the following fields:
 
-6. Select **Run All** to create a new warehouse (`gx_wh`) for the GX Agent.
+    - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.
+    
+    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique. If you use the same name for multiple Data Assets, each Data Asset must be associated with a unique Data Source.
 
-7. Copy the following code and paste it into the SQL worksheet:
+    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 
```

Comment:
Same as in the PostgreSQL guide:
```suggestion
    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the database. 
```

Reply:
@Rachel-Reverie the text has been updated.